x/group/keeper/msg_server.go

```diff
@@ -801,7 +777,7 @@ func (k Keeper) Exec(goCtx context.Context, req *group.MsgExec) (*group.MsgExecR
 		return nil, err
 	}
 
-	return res, nil
+	return &group.MsgExecResponse{}, nil
```

Comment:
Should we also add the result in the response?

Reply:
The result of executing the proposal? Yes I think that would make sense